MekHQ/src/mekhq/campaign/parts/equipment/LargeCraftAmmoBin.java

```diff
@@ -139,20 +138,20 @@ public double getTonnage() {
     }
 
     public double getCapacity() {
-        return capacity;
+        return size;
     }
 
     public void setCapacity(double capacity) {
-        this.capacity = capacity;
+        this.size = capacity;
     }
```

Comment:
Why even keep these, or at least not deprecate them?
Just temporary cause you don't want to make the change now to remove them? Or is it because they is no setter for size?

Reply:
I considered that but in the context of a LargeCraftAmmoBin I think the getCapacity() alias is more intuitive. I deprecated Mounted::getCapacity because the concept of capacity doesn't make sense for Mounted in every context.